manifests/config.pp

```diff
@@ -26,6 +22,16 @@
   }
 
   if $_real_mode {
+    if $facts['os']['family'] == 'Debian' and !$facts['selinux'] {
+      # Debian-based OSes also need to change the kernel boot parameters in the
+      # appropriate version of GRUB.
+      # See: https://wiki.debian.org/SELinux/Setup.
+      exec { 'activate-selinux':
```

Comment:
I'm not sure if this is the best approach. Puppet will execute this resource until the node got rebooted.

Reply:
Yes, this was a trade-off between robustness and idempotency-until-reboot.

I've rewritten the condition to check one of the immediately visible side-effects of the `activate-selinux` script instead.